releasenotes/notes/python-3-8-c66cde69f972eb1e.yaml

```
@@ -0,0 +1,14 @@
+# Each section from every release note is combined when the
+# CHANGELOG.rst is rendered. Therefore, the text should be worded in a way
+# that does not depend on any information available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Upgrade embedded Python 3 to 3.8.1
+    Link to Python 3.8 changelog: https://docs.python.org/3/whatsnew/3.8.html
+    Agent v6 will continue to ship both Python 2 & 3, respectively version 2.7.17 (unchanged) and 3.8.1
```

the container images of v6 only ship python 2, so we have to reword this 🤔 

How about:

```suggestion
    Note that the Python 2 version shipped in Agent v6 continues to be version 2.7.17 (unchanged).
```

Also, be careful about formatting because I think the reST format will ignore single newlines, so you'll want to separate lines with double newlines so that they actually appear as separate lines. (not sure if I'm being clear)

---

That's clear. I'm committing your suggestion and pushing the newlines changes in another commit.